src/utils/app.utils.js

```diff
@@ -75,6 +76,36 @@ export const getMissingAmountForFreeDelivery = (amountForFreeDelivery, cartTotal
   return ` ${(amountForFreeDelivery - cartTotal).toFixed(2)}`;
 };
 
+export const handleFileUpload = async (e, setTempImageUrl, setObjImageToUpload) => {
+  const resizeFile = (file) =>
+    new Promise((resolve) => {
+      ImageResizer.imageFileResizer(
+        file,
+        300, // new maxWidth
+        300, // new maxHheight
+        'JPEG', // output format
+        60, // quality (optional)
+        0, // rotation (optional)
+        (uri) => {
+          resolve(uri);
+        },
+        'base64' // output type (optional)
+      );
+    });
+
+  try {
+    if (e.target.files.lenght !== 0) {
```

Comment:
I think there is a typo here and that it should be `length`

Reply:
Ops...!